src/CMakeLists.txt

```diff
@@ -224,7 +222,7 @@ add_subdirectory(cppcache)
 add_subdirectory(cryptoimpl)
 add_subdirectory(dhimpl)
 add_subdirectory(sqliteimpl)
-add_subdirectory(gfcpp)
+add_subdirectory(getversion)
```

Comment:
I'm totally cool with deleting it. I didn't because it's been part of the install; some users may rely on it to determine version?